### PR TITLE
connects AlbumPage components using default redux state in preparation for MeteorLoader integration

### PR DESCRIPTION
**Pull request by rosscreighton**

### CODE REVIEW

File: imports/ui/components/FrostedPane/FrostedPane.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, { PropTypes } from 'react';
 
-export default function FrostedPane({ children }) {
+export function FrostedPane({ children }) {
```
Comment (sameckmeier) [inline imports/ui/components/FrostedPane/FrostedPane.jsx]: 👍 this is great. happy that we're on the same page with this. testing connected components was one of the items i wanted to talk about, but looks like we might not need to.

File: imports/ui/components/Review/Review.tests.jsx [hunk 2]
```diff
@@ -42,6 +42,6 @@ describe('<Review />', function () {
   });
 
   it('renders the review text', function () {
-    chai.expect(wrapper.contains(<TruncatedText wordCount={80} text={textValue} />)).to.equal(true);
+    chai.expect(wrapper.find('.Review-text').text()).to.contain(textValue);
```
Comment (sameckmeier) [inline imports/ui/components/Review/Review.tests.jsx]: 👍

File: imports/redux/store.js [hunk 3]
```diff
@@ -1,7 +1,10 @@
...
 import ReduxThunk from 'redux-thunk';
-import reducers from './reducers'
+import reducers from './reducers';
 
 const middleware = [ReduxThunk];
 
-export default createStore(reducers, {}, applyMiddleware(...middleware));
+export default createStore(reducers, {}, compose(
+  applyMiddleware(...middleware),
+  typeof window !== 'undefined' && window.devToolsExtension ? window.devToolsExtension() : f => f
```
Comment (sameckmeier) [inline imports/redux/store.js]: is this the integration for the chrome ext?

File: imports/ui/components/AlbumPage/AlbumPage.jsx [hunk 4]
```diff
@@ -1,32 +1,40 @@
...
+  return (
+    <div className="AlbumPage">
+      <PageHeader>
+        <HeaderContent />
+      </PageHeader>
+      <div className="AlbumPage-body">
+        <SectionHeader>
+          Reviews
+        </SectionHeader>
+        {reviews.map((review, i) => (
```
Comment (sameckmeier) [inline imports/ui/components/AlbumPage/AlbumPage.jsx]: what are your thoughts on having a Feed component? for sake of separating out responsibilities, i think that Page components shouldnt be passed any data, just params and dispatch; they should hydrate the store and render the layout a la meteorloader.

File: imports/ui/components/AlbumPage/AlbumPage.jsx [hunk 5]
```diff
@@ -11,11 +11,18 @@ import HeaderContent from '../HeaderContent';
 
 function AlbumPage({
   reviews,
+  artist,
```
Comment (sameckmeier) [inline imports/ui/components/AlbumPage/AlbumPage.jsx]: comment about reviews applies here too, but im guessing that this is all just for testing purposes after seeing this

File: imports/redux/store.js (hunk 3 above)
Comment (rosscreighton) [inline imports/redux/store.js]: Yezzir

File: imports/ui/components/AlbumPage/AlbumPage.jsx (hunk 4 above)
Comment (rosscreighton) [inline imports/ui/components/AlbumPage/AlbumPage.jsx]: Yeah I was thinking the same thing. I added a ticket to trello to talk about this as soon as I opened the PR. Glad that you caught it too. There are some downsides. Let's definitely discuss tomorrow.

**Pull request merged**